resources/zib/zib-wounds.WoundImage.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-wounds.WoundImage" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-wounds.WoundImage" />
+  <name value="ZibWoundsWoundImage" />
+  <title value="zib wounds.WoundImage" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A photo of the wound, burn wound or decubitus wound as visual information." />
+  <purpose value="This DocumentReference resource represents the WoundImage concept from Dutch zibs ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) [Burnwound v3.4(2020EN)](https://zibs.nl/wiki/Burnwound-v3.4(2020EN)), [PressureUlcer v3.3(2020EN)](https://zibs.nl/wiki/PressureUlcer-v3.4(2020EN)) and [Wound v3.3(2020EN)](https://zibs.nl/wiki/Wound-v3.3(2020EN))." />
```

Comment:
Please align the purpose to the format I used in https://github.com/Nictiz/Nictiz-R4-zib2020/pull/407.

Reply:
Fix